packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.spec.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { expect } from 'chai'
+import Meta, { Default, NoSubject } from './FlipbookViewer.stories'
+import { composeStory } from '@storybook/testing-react'
+import userEvent from '@testing-library/user-event'
+
+describe('Component > FlipbookViewer', function () {
+  describe('with a valid subject', function () {
+    const DefaultStory = composeStory(Default, Meta)
+
+    it('should render the correct number of thumbnnails', function () {
+      const { getAllByRole } = render(<DefaultStory />)
+      const thumbnailButtons = getAllByRole('tab')
+      expect(thumbnailButtons).to.have.length(4)
+    })
+
+    it('should highlight the active frame thumbnail with a border', function () {
+      const { getAllByRole } = render(<DefaultStory />)
+      const thumbnailButtons = getAllByRole('tab')
+      const { border } = window.getComputedStyle(thumbnailButtons[0])
+      expect(border).to.equal('2px solid #f0b200') // neutral-2 in theme
+    })
+
+    it('should have previous and next buttons', function () {
+      const { getByText } = render(<DefaultStory />)
+      const nextButton = getByText('SubjectViewer.MultiFrameViewer.FrameCarousel.nextFrameLabel')
+      const prevButton = getByText('SubjectViewer.MultiFrameViewer.FrameCarousel.previousFrameLabel')
+      expect(nextButton).exists()
+      expect(prevButton).exists()
+    })
+
+    it('should handle changing the current frame via thumbnail', async function () {
+      const user = userEvent.setup({ delay: null })
+
+      const { getAllByRole, rerender } = render(<DefaultStory />)
+      const thumbnailButtons = getAllByRole('tab')
+      const buttonStyle = window.getComputedStyle(thumbnailButtons[0])
+      expect(buttonStyle.border).to.equal('2px solid #f0b200') // neutral-2 in theme
+
+      await user.pointer({
+        keys: '[MouseLeft]',
+        target: thumbnailButtons[1]
+      })
+
+      rerender(<DefaultStory />)
+      const newThumbnailButtons = getAllByRole('tab')
+      const newButtonStyle = window.getComputedStyle(newThumbnailButtons[1])
+      expect(newButtonStyle.border).to.equal('2px solid #f0b200')
+    })
+
+    it('should handle using arrow keys on the tablist', async function () {
+      const user = userEvent.setup({ delay: null })
+
+      const { getAllByRole, rerender } = render(<DefaultStory />)
+      const thumbnailButtons = getAllByRole('tab')
+      expect(thumbnailButtons[0].tabIndex).to.equal(0)
+
+      await user.tab() // focus on Previous Button
+      await user.tab() // focus on tablist
+      await user.keyboard('{ArrowRight}')
```

Comment:
I want to test usage of an arrow key while focused in the tablist. Is there a better way to focus on the desired element instead of calling `user.tab()` multiple times? For instance, the play button and speed control will eventually be before the tablist in the DOM, which would mean adding more `user.tab()` calls to this unit test.

Reply:
`thumbnailButtons[0].focus()` should work.

Reply:
yes! thank you